src/utils/params.js

```diff
@@ -30,17 +30,22 @@ export const defaultParams = {
   stopKeydownPropagation: true,
   keydownListenerCapture: false,
   showConfirmButton: true,
+  showDenyButton: false,
   showCancelButton: false,
   preConfirm: undefined,
   confirmButtonText: 'OK',
   confirmButtonAriaLabel: '',
   confirmButtonColor: undefined,
+  denyButtonText: 'No',
```

Comment:
Changed to `'Deny'`

Reply:
"No" was better IMO because 
- to have "Yes" / "No" (surely the most frequent choice for apps hard-coded with English), only need to change `confirmButtonText`
- if developer doesn't bother setting *any* button text when firing a confirm/deny (yes/no) swal (for example, asking something like "Save changes?"), I think "OK" / "No" is nicer than "OK" / "Deny"..

Reply:
Reverted to "No" 👍 

Can't come up with anything better than that.

Reply:
Any suggestions from other reviewers would be welcome 🙇